ppr-ui/src/composables/mhrRegistration/home-owner.ts

```diff
@@ -1,9 +1,33 @@
-import { computed } from '@vue/composition-api'
+import {
+  MhrRegistrationHomeOwnerGroupIF,
+  MhrRegistrationHomeOwnersIF
+} from '@/interfaces'
+import '@/utils/use-composition-api'
+
+import { ref, computed, readonly, watch } from '@vue/composition-api'
+import { useActions, useGetters } from 'vuex-composition-helpers'
+import { find, remove, set, findIndex } from 'lodash'
+import { Watch } from 'vue-property-decorator'
+
+const DEFAULT_GROUP_ID = '1'
```

Comment:
Noticed this was defined on the AddEditHomeOwner as well. Intentional?

Reply:
There was a Default Owner ID, just a starting ID for Owners and Groups. All Owners would go to Group 1 automatically by default.